satellite/orders/service.go

```diff
@@ -475,36 +484,43 @@ func (service *Service) CreateAuditOrderLimit(ctx context.Context, auditor *iden
 
 	err = service.saveSerial(ctx, serialNumber, bucketID, orderExpiration)
 	if err != nil {
-		return nil, Error.Wrap(err)
+		return nil, storj.PiecePrivateKey{}, Error.Wrap(err)
 	}
 
 	projectID, bucketName, err := SplitBucketID(bucketID)
 	if err != nil {
-		return limit, err
+		return limit, piecePrivateKey, Error.Wrap(err)
```

Comment:
Why do we return `piecePrivateKey` in this error case, but not in the other error cases? Is there something special going on here?

Reply:
yeah, I'll make it empty